src/components/Common/AvatarEditModal.tsx

```
@@ -286,69 +424,92 @@
             {!isCameraOpen ? (
               <>
-                {preview || imageUrl ? (
+                {preview ? (
                   <>
-                    <div className="flex h-[30vh] md:h-[75vh] w-full items-center justify-center rounded-lg border border-secondary-200">
-                      <img
-                        src={
+                    <div className="relative w-full h-[400px]">
+                      <Cropper
+                        image={
                           preview && preview.startsWith("blob:")
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)